uploader.json

```diff
@@ -882,6 +882,12 @@
             "pattern": [],
             "symbolic_name": "systool_b_scsi_v"
         },
+        {
+            "command": "/usr/bin/teamdctl  config dump",
```

Comment:
@rasrivas-redhat will this command work if the iface argument is appended to the end of this command?  That is the way pre_commands work, they are appended to the end of the command string.

Reply:
@bfahr - Apology for the late response as I was on PTO on 22 March.

1. This command will not work if iface argument is appended to the end of this command
2. We already have "**teamdctl_state_dump**" command, which works in the same way, accordingly I have updated the code for "**teamdctl_config_dump**", please let me know if any changes are required.